metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -304,7 +306,8 @@ void main() {
         notifier.push(DefaultRoutes.projectGroups);
         notifier.pop();
 
-        verify(pageParametersFactory.create(currentConfiguration)).called(once);
+        verify(pageParametersFactory.create(currentConfiguration))
+            .called(equals(2));
```

Comment:
✅ 
It seems like here we are expecting to get only one call of the `pageParametersFactory.create(currentConfiguration)`.